README.md

```diff
@@ -35,6 +35,15 @@ also passing it your key encryption key and optionally the database to insert in
   (encrypted-fields key-encryption-key :number :expiration-date-month :expiration-date-year))
 ```
 
+When performing updates on entities with encrypted fields, use `korma-encrypted/update-encrypted-entity` instead of the standard `korma/update` function.  This ensures data encryption key foriegn keys stored on encrypted entities are preserved on updates.
+
+```clojure
+
+(update-encrypted-entity credit-card-with-encrypted-fields
+                         primary-key-of-card-to-update
+                         map-of-fields-to-update)
```

Comment:
Yeah we talked about this quite a while yesterday. 

1. We tried to have them pass a `korma/set-fields` instead of a map, but internally we need a way to modify that set-fields before it even gets to the `prepare` callback so that we can lookup the existing record's data_encryption_key_fk. And it doesn't look like the thing that comes from `set-fields` can be modified (we may have missed that though, so if you know anything offhand let us know). So we went with a map instead of that.
2. We went with a pk rather than a `where` because we explicitly  wanted to make it clear that this will only work for a single record. With the normal update, you could pass in a `where` clause that would update any number of records (let's say `n`), and to have encrypted-fields support that we would have to load all of the records (one query), and then run a separate update statement for each record (`n` queries). So suddenly the method looks very similar to the normal update method which is ~constant time, but actually is horrifically inefficient. So we decided that for now we would just support a single update at a time, and made the method signature purposefully different so that it's super obvious to the user what is actually going to happen.

Not sure if you have thoughts about any of that?